src/pages/OnboardingPage/OnboardingPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from "react";
+
+import GenericPage from "../GenericPage";
+import { RoleSelectOnboard } from "../../components/RoleSelectOnboard/RoleSelectOnboard";
+import { RegistrationOnboarding } from "../../components/RegistrationOnboarding/RegistrationOnboarding";
+
+export interface OnboardingPageProps {
+    footerLinks: {
+        summary: string
+        links: { linkContent: string; linkUrl: string }[]
+        id: string
+      }[],
+    isChecked: boolean,
+    isPrimary: boolean,
+}
+
+export const OnboardingPage: React.FC<OnboardingPageProps> = ({
+    footerLinks, 
+    isChecked,
+    isPrimary,
+}) => {
+    const [isClicked, setIsClicked] = useState(false);
```

Comment:
lets rename it to be less generic (selectedRole) and accept selected role in handle click as a string.

Reply:
Role value will have to be sent to the backend in registration request